src/main/java/edu/harvard/iq/dataverse/datasetutility/AddReplaceFileHelper.java

```diff
@@ -5,16 +5,11 @@
  */
 package edu.harvard.iq.dataverse.datasetutility;
 
-import edu.harvard.iq.dataverse.DataFile;
+import edu.harvard.iq.dataverse.*;
```

Comment:
We generally avoid wildcard imports. If you feel like changing it back, great. Otherwise, no big deal.

Reply:
@pdurbin - I will submit the corrections.